data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/heals/heal_poison_slowed.cfg

```diff
@@ -20,7 +20,8 @@
             [effect]
                 apply_to = new_ability
                 [abilities]
-                    {TEST_ABILITY heals 4 ([affect_adjacent][/affect_adjacent]) ALLIES=yes}
+                    {TEST_ABILITY heals 4 ([affect_adjacent][/affect_adjacent]
+                    poison="slowed") ALLIES=yes}
```

Comment:
Line wrapping reduced the readability here

Reply:
Updated.